webapp/package-lock.json

```diff
@@ -227,6 +228,121 @@
         "yargs": "16.2.0"
       }
     },
+    "channels/node_modules/@mattermost/mmjstool": {
```

Comment:
i think this wasnt intentional change, but its fine i think we didnt update the lock file for a while

Reply:
nope, reran it and not there. I think I ran an odd i18n-extract command in a different directory or something.